src/Datadog.Trace/Agent/AgentWriterBuffer.cs

```diff
@@ -1,55 +0,0 @@
-using System;
-
-namespace Datadog.Trace.Agent
-{
-    internal class AgentWriterBuffer<T>
```

Comment:
Wow, this is one of the oldest files in the tracer, before my time. I tracked it own to commit 21df6e07559d880ce98c4f8bfddfa03041bc5816 in Oct 2017. cc @bmermet